build.gradle

```diff
@@ -52,3 +52,104 @@ task replicateSources << {
 		}
 	}
 }
+
+configurations {
+	bnd
+}
+
+dependencies {
+	bnd "biz.aQute.bnd:bnd:2.4.0"
+}
+
+repositories {
+	mavenLocal()
+
+	mavenCentral()
+}
+
+task jarBndTools(type: GradleBuild) {
+	dir = file("bndtools")
+	tasks = ["jar"]
+}
+
+task jarGradle(type: GradleBuild) {
+	dir = file("gradle")
+	tasks = ["jar"]
+}
+
+task jarLiferayGradle(type: GradleBuild) {
+	dir = file("liferay-gradle")
+	tasks = ["jar"]
+}
+
+task diff << {}
+
+diff.dependsOn diffGradle
+diff.dependsOn diffBndTools
+diff.dependsOn diffMaven
+
+task diffGradle(dependsOn: [jarGradle, jarLiferayGradle]) << {
+	liferayGradleOutputFiles.each {
+		liferayGradleOutputFile ->
+
+		def gradleOutputFile = gradleOutputFiles.find {
+			it.name == liferayGradleOutputFile.name
+		}
+
+		println "*******************************************************"
+		println "*******************************************************"
+		println "*******************************************************"
+
+		javaexec {
```

Comment:
Can you try to capture this output?  And them process the output?  Check for errors or unexpected differences?